subjects/binary_search/README.md

```diff
@@ -0,0 +1,45 @@
+## binary_search
+
+### Instructions
+
+Write a function called `binary_search` that performs a binary search on a sorted list of integers. The function should take the following arguments:
+
+```rust
+pub fn binary_search(sorted_list: &[i32], target: i32) -> Option<usize> {
+	// Your code goes here
+}
+```
+
+- `sorted_list: &[i32]`: A reference to a sorted list of integers to search within.
```

Comment:
This list should follow immediately the text "The function should take the following arguments:"